### DISCUSSION TITLE
在编译带有Qt项目时编译失败

### DISCUSSION THREAD

**Discussion by suzhuorui** [xmake-io/xmake#3067] [Q&A]
在编译带有Qt项目时编译失败，提示找不到**ui_xxxx.h**### 
在重复的多次xmake后，编译成功
这个估计是个bug,希望作者大佬修复一下

**Comment by waruqi** [comment]
有问题，提 issues，并提供详细 xmake.lua 配置， logs，版本。。。就来一句找不到。我还能凭空猜出来？

**Reply by augustheart** [reply]
估计还是哪儿写得不太对。不过说实话我也碰到类似的情况。前段时间写soui的编译的时候遇到过（已经由于兴趣的转移放弃了）。但是确实就是没法定位。不输出日志时随机抛出错误，输出日志时它居然就不出错了！！！
（我有个不靠谱盲猜，我的情况可能和我在package里面用了set_kind("static")或是其它不应该在package里面用但是又不会报错的接口有关。）
@waruqi package里面应该是不能用static的吧？

**Reply by waruqi** [reply]
package 默认就是 library ，没有 static/shared kind ，不要去设置 set_kind("static")
